lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -627,3 +629,16 @@ func getNetworkSentryConfig(network *configurator.Network) *lte_mconfig.SentryCo
 		UrlPython:    string(sentryConfig.URLPython),
 	}
 }
+
+// getSyncInterval takes network-wide subscriber_db sync interval and overrides it if also set for gateway.
+// If sync interval is unset for both network and gateway, a default is read from lte/cloud/configs/subscriberdb.yml
+func getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, gwEpc *lte_models.GatewayEpcConfigs) uint32 {
+	syncInterval := gwEpc.SubscriberdbSyncInterval
+	if syncInterval == 0 {
+		syncInterval = nwEpc.SubscriberdbSyncInterval
+	}
+	if syncInterval == 0 {
+		return subscriberdb.MustGetServiceConfig().SyncInterval
+	}
+	return uint32(syncInterval)
```

Comment:
Refrain from reassigning a variable where not necessary. So either use early returns, or a switch statement, or just use what you have but use new var names that are descriptive of their contents